libs/dev_only_scripts/configToDb.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+require_once '/var/www/html/libs/php-commons/Db.php';
+
+$dir = '/var/www/html';
+
+checkTemplate($dir);
+
+function checkTemplate($folder) {
+    //if (is_dir($folder . '/.svn')) {
+    if (is_dir($folder . '/sources')) {
+        if (file_exists($folder . '/sources/DbConfig.php')) {
+            // we are in a portal
+            $portal = str_replace('/var/www/html', '', $folder);
+
+            // instantiate a national_leaf_launchpad db
+            $launchpad_db = new \Leaf\Db(getenv('DATABASE_HOST'), getenv('DATABASE_USERNAME'), getenv('DATABASE_PASSWORD'), 'national_leaf_launchpad');
+
+            // get the db name for this portal
+            $vars = array(':site_path' => $portal);
+            $sql = 'SELECT portal_database
+                    FROM sites
+                    WHERE site_path= BINARY :site_path';
+
+            $site_paths = $launchpad_db->prepared_query($sql, $vars)[0];
+
+            // instantiate a portal db
+            $db =  new \Leaf\Db(getenv('DATABASE_HOST'), getenv('DATABASE_USERNAME'), getenv('DATABASE_PASSWORD'), $site_paths['portal_database']);
+
+            // instantiate a Config.php
+            require_once $folder . '/sources/Config.php';
+            $config = new Portal\Config();
+
+            // populate db with config data
+            $sql = 'INSERT INTO `settings` (`setting`, `data`)
+                    VALUES (:setting, :data)
+                    ON DUPLICATE KEY UPDATE `setting`=:setting';
+
+            $vars = array(':setting' => 'heading',
+                    ':data' => $config->title);
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'subHeading',
+                    ':data' => $config->city);

```

Comment:
Same here, we don't want to overwrite the data users have set already since this is an empty variable on the configs. Either make a check for empty data to then input the data or just leave it.

Reply:
same as above reply.